main.js

```diff
@@ -0,0 +1,252 @@
+      game.board[0][2] = this.symbol;
+      counter++;
+      // this.array.push(3);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play4() {
+    if (checkPositions().position4 != "X" && checkPositions().position4 != "O") {
+      game.board[1][0] = this.symbol;
+      counter++;
+      // this.array.push(4);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play5() {
+    if (checkPositions().position5 != "X" && checkPositions().position5 != "O") {
+      game.board[1][1] = this.symbol;
+      counter++;
+      // this.array.push(5);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play6() {
+    if (checkPositions().position6 != "X" && checkPositions().position6 != "O") {
+      game.board[1][2] = this.symbol;
+      counter++;
+      // this.array.push(6);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play7() {
+    if (checkPositions().position7 != "X" && checkPositions().position7 != "O") {
+      game.board[2][0] = this.symbol;
+      counter++;
+      // this.array.push(7);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play8() {
+    if (checkPositions().position8 != "X" && checkPositions().position8 != "O") {
+      game.board[2][1] = this.symbol;
+      counter++;
+      // this.array.push(8);
+      game.checkWinner(game.board);
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play9() {
+    if (checkPositions().position9 != "X" && checkPositions().position9 != "O") {
+      game.board[2][2] = this.symbol;
+      counter++;
+      // this.array.push(9);
+      game.checkWinner(game.board);
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+
+};
+
+var gameState = function() {
+    console.log("This is the current game board: " + "\n" + game.board[0] + "\n" + game.board[1] + "\n" + game.board[2]);
+};
+
+var moveTaken = function(){
+  console.log("That move is already taken!")
+}
+
+var checkPositions = function() {
+  var position1 = game.board[0][0];
+  var position2 = game.board[0][1];
+  var position3 = game.board[0][2];
+  var position4 = game.board[1][0];
+  var position5 = game.board[1][1];
+  var position6 = game.board[1][2];
+  var position7 = game.board[2][0];
+  var position8 = game.board[2][1];
+  var position9 = game.board[2][2];
+
+  return {
+    position1: position1,
+    position2: position2,
+    position3: position3,
+    position4: position4,
+    position5: position5,
+    position6: position6,
+    position7: position7,
+    position8: position8,
+    position9: position9
+  }
+}
+// Shortened win checker not working.
+// var checkWin = function(winner, array) {
+//   return array.some(val => winner.indexOf(val) === -1)};
+
+var counter = 0;
```

Comment:
Why define this way down here? Is this being used to count the number of terms? I think there is a better way than a global variable like this.